stubs/ext-ds.stub

```diff
@@ -17,9 +17,9 @@ use UnderflowException;
 interface Collection extends Traversable, Countable, JsonSerializable
 {
 	/**
-	 * @return Collection<TKey, TValue>
+	 * @return static<TKey, TValue>
```

Comment:
`static<TKey, TValue>` isn't really supported right now, but in  this case you can just do `static`.

Reply:
I'm just experimenting for now. I don't even understand the problem fully yet. ;-)